.github/workflows/build.yml

```diff
@@ -66,14 +67,49 @@ jobs:
             ${{ runner.os }}-gradle-
       - name: Build
         run: ./gradlew build
-      - name: Generate Jacoco report
-        run: ./gradlew jacocoTestReport
+
+  test:
```

Comment:
Let's name it: Integration Tests

Reply:
I think "test" is better because we are running both the integration and the unit tests

Reply:
Yes, this is for both the integration and the unit tests